src/helpers/menuConfigs.ts

```diff
@@ -86,18 +87,43 @@ function mapToLocalIcon<T extends { icon: string }>(
   }));
 }
 
-export const bundlerPackages = bundlerOptions.map(b => b.package);
+export const bundlerPackages = bundlerOptions.map((b) => b.package);
 
 export function getResourcePath(panel: vscode.WebviewPanel, baseUriResources: string, fileName: string) {
   return panel.webview.asWebviewUri(vscode.Uri.file(join(baseUriResources, fileName)));
 }
 
+export function getToolkitUri(webview: vscode.Webview, extensionUri: vscode.Uri) {
+  const pathList = ['node_modules', '@vscode', 'webview-ui-toolkit', 'dist', 'toolkit.min.js'];
```

Comment:
No.

Pull the latest changes. You cannot reference anything from node_modules as node_modules is not there when the extension is installed.